notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-095_settling_big_vibrations.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.        correctedVariableCheck2 = np.square(correctedVariableCheck)
</pre><p>It looks to me that the comments in lins 60-62 do not correspond to the code at line 63-65. Could you please update the comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Deprecated, this notebook has been removed.